packs/icinga2/sensors/lib/client.py

```diff
@@ -0,0 +1,136 @@
+import pycurl, json
+import urllib
+import signal
+import sys
+import time
+
+POST_PARAMS = {}
+
+class Client:
```

Comment:
Same here and it also looks like a lot of code is duplicated. Can rid of most of it by switching to requests (mall other packs use requests so there should be plenty of usage examples around) :)